### PR TITLE
fix: Fix to build s1sim application and traffic generator on aarch64

### PR DESCRIPTION
**Pull request by m-govind**
Signed-off-by: mgovind <govindarajan.mohandoss@arm.com>

## Title
Enabling build in aarch64

## Summary
This fix is essential to enable containerized S1AP test in Magma on aarch64. The Makefiles under TestCntlrApp and Trfgen are modified to enable aarch64 build. Also, missing packages like libpcap and libiperf are uploaded for aarch64.

## Test plan
1. On both x86 and aarch64:
    a. ~/Trfgen/build
        make clean & make
    b. ~/TestCntlrApp/build
        make cleanall & make

### CODE REVIEW

**Comment by rsarwad** [comment]
Hi @m-govind, changes looks good to me
Could you update summary and test plan

File: TestCntlrApp/build/fw.mak [hunk 1]
```diff
@@ -69,7 +69,12 @@ BIN_DIR=../bin
 LIB_DIR=../lib
 
 # pcap archive name
-PCAP_ARC=libpcap-64bit.a
+UNAME_M := $(shell uname -m)
```
Comment (VinashakAnkitAman) [inline TestCntlrApp/build/fw.mak:72]: We are evaluating this UNAME_M twice in this file. We can evaluate once and use at both the places for comparing it

Comment (m-govind) [reply TestCntlrApp/build/fw.mak:72]: Done.

**Comment by m-govind** [comment]
> Hi @m-govind, changes looks good to me Could you update summary and test plan

Done

File: TestCntlrApp/build/fw.mak (hunk 1 above)
Comment (VinashakAnkitAman) [reply TestCntlrApp/build/fw.mak:72]: LGTM. Just need to confirm if sanity is passing over this PR

**Review by VinashakAnkitAman — APPROVED** [review]
> LGTM. Just need to confirm if sanity is passing over this PR

The sanity is passing over the PR changes on my local 3 VM setup with
Host: Ubuntu 20.04.3 LTS (x86_64)
Magma-Test VM: Ubuntu 20.04.1 LTS (x86_64)

[Sanity_Log_S1APTester_PR_76.txt.txt](https://github.com/magma/S1APTester/files/9288688/Sanity_Log_S1APTester_PR_76.txt.txt)

**Pull request merged**